src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(10, 2)).toEqual([5, 5]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(10, 1)).toEqual([10]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(7, 3)).toEqual([2, 2, 3]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
-
+  expect(splitInteger(1, 0)).toEqual([]);
```

Comment:
The test case `splitInteger(1, 0)` is incorrect because `numberOfParts` should always be a positive integer. This test case should be revised or removed as it does not align with the task description.